spec/duration.html

```diff
@@ -1370,6 +1363,20 @@ <h1>RoundDuration ( _years_, _months_, _weeks_, _days_, _hours_, _minutes_, _sec
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-temporal-rounddurationcomplete" aoid="RoundDurationComplete">
+      <h1>RoundDurationComplete ( _duration_, _smallestUnit_, _largestUnit_, _relativeTo_, _roundingIncrement_, _roundingMode_ )</h1>
```

Comment:
I'm trying to make sure we add a `<p>` with a blurb about each abstract operation going forward, and possibly information about the types of the arguments. Would you mind adding one here?